### DISCUSSION TITLE
What is `atl06_quality_summary` returned by atl06s?

### DISCUSSION THREAD

**Discussion by scottyhq** [SlideRuleEarth/sliderule#441] [General]
https://slideruleearth.io/web/rtd/api_reference/icesat2.html#atl06s

I was looking at the docs here, but think those return values are specific to `atl06` and `atl06p`?
https://slideruleearth.io/web/rtd/user_guide/ICESat-2.html#elevations-atl06

**Comment by scottyhq** [comment]
Ah, just found https://nsidc.org/sites/default/files/icesat2_atl06_data_dict_v003_0.pdf 

which says: 

> The ATL06_quality_summary parameter indicates the best-quality subset of all ATL06 data. A zero in this parameter implies that no data-quality tests have found a problem with the segment, a one implies that some potential problem has been found.

> Users who select only segments with zero values for this flag can be relatively certain of obtaining high-quality data, but will likely miss a significant fraction of usable data, particularly in cloudy, rough, or low-surface-reflectance conditions.

> Flag Values: ['0', '1'].  Flag Meanings: ['best_quality', 'potential_problem']

**Comment by dshean** [comment]
Thanks Scott.  Seems like we should note this somewhere in the SlideRule doc, and link to the ATL06 data dictionary.  Same with the GEDI quality flags: https://github.com/orgs/SlideRuleEarth/discussions/442.  
Would be great to offer more guidance for users on how to interpret and filter the SlideRule returns for IS2 and GEDI.  Maybe a separate doc section on how to review and use product quality flags, sigma/rms values, difference relative to reference DEM?   I was talking to Michelle Hofton about this last week - the GEDI team has optimized filters for different applications/locations, and maybe we can try to implement some of that in SlideRule.
Experienced altimetry users will have their own approaches, but for most beginners and/or modelers who just want points they can trust, this would likely be helpful.
